### ISSUE TITLE
program hangs router.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#118]
```
What steps will reproduce the problem?
1.Load Program
2.default settings, or make changes
3.after about 30 seconds, dns fails during web site lookup, while "Checking
Nameserver Availability" is running. If allowed to complete, shows no name
servers are responding. 

What version of the product are you using? On what operating system?
1.2, Windows 7 64 bit. 

Please provide any additional information below.
A similar dns benchmarking program does not result in complete dns lookup
failure on my router. After running the test, I have to physically power
cycle the router. Perhaps it's the router (Netgear WGR614v6).

```

Original issue reported on code.google.com by `lowered9...@gmail.com` on 10 May 2010 at 8:03

**Comment by GoogleCodeExporter** [comment]
```
Just wanted to add, maybe 40 threads is too many (max open connections) in 
windows 7.
Perhaps this could be a user adjustable number?
```

Original comment by `lowered9...@gmail.com` on 10 May 2010 at 8:05

**Comment by GoogleCodeExporter** [comment]
```
Can you try the newly released namebench 1.3-BETA1 to see if it resolves this 
bug? I have added a dropdown to 
force the health tests to be run in a slower mode (10 threads). 
```

Original comment by `helixblue` on 20 May 2010 at 3:20

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3 (added an option for unstable networks).

```

Original comment by `helixblue` on 31 May 2010 at 6:44
- Changed state: **Fixed**
